parquet-common/src/main/java/org/apache/parquet/bytes/ByteBufferInputStream.java

```diff
@@ -138,6 +134,18 @@ public int read(byte[] b, int off, int len) throws IOException {
     return delegate.read(b, off, len);
   }
 
+  public int read(byte[] b) throws IOException {
+    return read(b, 0, b.length);
+  }
+
+  public void readFully(byte b[]) throws IOException {
```

Comment:
Why we need to be different between line 137 ' byte[] b' and 141 ' byte b[]'?

Reply:
This is almost certainly the result of a copy/paste of the method signature, and I just didn't notice. I went ahead and fixed it.